app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,17 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self,
+                 nickname: str,
+                 musical_instrument: str,
+                 bow_level: int) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._bow_level = bow_level
+
+    def player_info(self) -> str:
+        return (f"Elf ranger {self.nickname}. "
+                f"{self.nickname} has bow of the {self._bow_level} level")

```

Comment:
In the `player_info` method, the phrase 'has bow of the {self._bow_level} level' could be improved for clarity. Consider rephrasing it to 'has a bow with a level of {self._bow_level}' for better readability.